web/src/stores/app-store.ts

```diff
@@ -225,7 +226,7 @@ export const useAppStore = defineStore('app', () => {
     getLastEdit,
     setLastSave,
     setLastEdit,
-    getFactories,
+    getFactories: () => factories.value,
```

Comment:
@nils-keller-dev Hmm I'd slightly disagree with this one for one primary reason, the current `getFactories()` method ensures that the data is correctly migrated. If we're able to refactor so that when the page loads that the data is fed through the current `getFactories()` method then referencing the value directly thereafter would work. I added the checks in `getFactories` so that it ensures data integrity despite whatever the source may be.

Reply:
Perhaps we create a new one called `initialLoad` or similar

Reply:
Added it in myself